mautic/mautic
# TPROD-340 - Migrate Tag Manager Views to twig

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


This PR does one main thing:

- Migrate the views for tag manager plugin to twig

#### Steps to test this PR:

##### Testing from and end-user/UI perspective

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. in the left side menu click on "tags"
3. Test every single functionality on this page. Creating/editing/viewing tags, filtering, pagination, etc. - all functionality should work exactly like it did before.


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.
